Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +13,449 @@ public struct GitHub: Decodable {
         case requestedReviewers = "requested_reviewers"
     }
 
-    // MARK: - Properties
-
-    public let issue: GitHubIssue
+    public let issue: Issue
 
-    public let pullRequest: GitHubPR
+    public let pullRequest: PR
 
-    public let commits: [GitHubCommit]
+    public let commits: [Commit]
 
-    public let reviews: [GitHubReview]
+    public let reviews: [Review]
 
-    public let requestedReviewers: GitHubRequestedReviewers
+    public let requestedReviewers: RequestedReviewers
 
     public internal(set) var api: Octokit!
 }
 
-// MARK: - GitHubPR
-
-public struct GitHubPR: Decodable, Equatable {
-    // MARK: - CodingKeys
-
-    enum CodingKeys: String, CodingKey {
-        case number
-        case title
-        case body
-        case user
-        case assignee
-        case assignees
-        case milestone
-        case additions
-        case deletions
-        case state
-        case head
-        case base
-        case isLocked = "locked"
-        case isMerged = "merged"
-        case createdAt = "created_at"
-        case updatedAt = "updated_at"
-        case closedAt = "closed_at"
-        case mergedAt = "merged_at"
-        case commitCount = "commits"
-        case commentCount = "comments"
-        case reviewCommentCount = "review_comments"
-        case changedFiles = "changed_files"
-        case htmlUrl = "html_url"
-    }
-
-    // MARK: - PullRequestState
-
-    public enum PullRequestState: String, Decodable {
-        case open
-        case closed
-        case merged
-        case locked
+extension GitHub {
+    public struct PR: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
```

Comment:
<!--
  0 failure: 
  1 warning:  Types should be n...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 31;
-->

- :warning: Types should be nested at most 1 level deep (`nesting`)